pkg/useragent/mongodb_test.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
```

Comment:
These tags allow separating out unit tests and integration tests.  `go test` will run unit tests by default and can run integration tests instead by using `go test --tags=integration ...`